### PR TITLE
created dashboard overview document

### PR DESCRIPTION
**Pull request by josep-fl**
Created dashboard overview page

### CODE REVIEW

File: site/content/docs/dashboards/dashboard-url.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+weight: 2
+menu:
+  docs:
+    parent: "Dashboards"
+aliases:
+- "/dashboards/dashboard-url/"
+- "/docs/dashboards/dashboard-url/"
+---
+
+Checkly dashboards are public dashboards available on every plan. Use them on your wall mounted TVs, on
```
[the remark this answers is not in the document]
Comment (ragog) [reply site/content/docs/dashboards/dashboard-url.md:12]: Maybe the URL & Custom Domain page should just get absorbed into Overview?

File: site/content/docs/dashboards/dashboard-url.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+shared monitors around the office or as a status page on your site.
+
+An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com).
+
+> Dashboards can be **embedded in iframes**.
```
Comment (ragog) [inline site/content/docs/dashboards/dashboard-url.md:17]: Let's move this down, below the next phrase ("You can create multiple...").

File: site/content/docs/dashboards/_index.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 ---
-title: Url and Custom Domain
+title: Overview
```
Comment (ragog) [inline site/content/docs/dashboards/_index.md:2]: This page should show at least an example screenshot of a complete dashboard. We are talking about dashboards in this section without ever showing what they look like.

**Review by ragog — CHANGES_REQUESTED** [review]
@josep-fl please see my comments. I have added some directions but I expect you to go through these pages in-depth and propose improvements on your own.

File: site/content/docs/dashboards/_index.md [hunk 4]
```diff
@@ -1,46 +1,41 @@
...
-Typically a company name works best.
+An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com) as shown in the screenshot below.
 
-![custom url for public dashboard](/docs/images/dashboards/custom_url.png)
+![Dashboard example](/docs/images/dashboards/dashboard_example.png)
 
+## Adding checks to your dashboard
 
-## Custom Domain
+In order to show specific checks in a dashboard, you will need to do this:
```
Comment (ragog) [inline site/content/docs/dashboards/_index.md:27]: @josep-fl this whole section still reads like a support ticket, can we improve it?

Comment (ragog) [reply site/content/docs/dashboards/_index.md:27]: Also, let's make it clear that all checks are automatically shown if we do not specify exactly which ones to include.

**Review by ragog — CHANGES_REQUESTED** [review]
@josep-fl getting there, just a couple of changes needed in the new section.

**Comment by ianaya89** [comment]
Looks good to me. Just one comment:
@josep-fl would be great to use [commitizen style](https://github.com/commitizen/cz-cli) for commits to keep consistency
image

File: site/content/docs/dashboards/_index.md [hunk 5]
```diff
@@ -24,18 +24,16 @@ An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com
 
 ## Adding checks to your dashboard
 
-In order to show specific checks in a dashboard, you will need to do this:
+By default, if no check is specified, all checks in the account will be shown in the dashboard. If instead you want to show show specific checks, this is how it's accomplished:
```
Comment (ragog) [inline site/content/docs/dashboards/_index.md:27]: Typo: "show show"

File: site/content/docs/dashboards/_index.md [hunk 6]
```diff
@@ -24,18 +24,16 @@ An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com
...
 
 1. First add tags to one or more checks you would like to show. This is done by editing the check(s). 
 
 ![Tags](/docs/images/dashboards/tags.png)
 
-Don't forget to press Enter to save the tag before you save your updated check!
+You'll need to press Enter to save the tag before you save your updated check!
```
Comment (ragog) [inline site/content/docs/dashboards/_index.md:33]: I think the 1. & 2. points mess up the page a bit with the indentation (it looks more confusing than it should). My 2 cents.

**Review by ragog — APPROVED** [review]
LGTM @josep-fl

**Pull request merged**